code/style.css

```diff
@@ -1,7 +1,212 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+}
 
-.active + .answer {
-  /* some styling here */
+/*** HEADER ***/
+
+header {
+  display: flex;
+  align-items: center;
+}
+
+video, img {
+  width: 100%;
+  filter: grayscale(100%); 
+} 
+
+@media (min-width: 1024px) { 
+  .img-tablet, .img-phone {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) { 
+  video, .img-phone {
+    display: none;
+  }
+}
+
+@media (max-width: 767px) { 
+  video, .img-tablet {
+    display: none;
+  }
+}
+
+h1 {
+  width: 100%;
+  position: absolute;
+  text-align: center;
+  color: white;
+  font-size: 22.1vw;
+
+  animation-name: lightning;
+  animation-duration: 0.1s;
+  animation-delay: 5s;
+  animation-iteration-count: 20;
+}
+
+@keyframes lightning {
+  from {color: white;}
+  to {color: yellow;}
+} 
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  h1 {
+    animation-name: slowyellow;
+    animation-duration: 10s;
+    animation-delay: 2s;
+    animation-direction: alternate;
+    animation-iteration-count: infinite; 
+  }
+}
+
+@keyframes slowyellow {
+  from {color: white;}
+  to {color: yellow;}
+} 
+
+@media (max-width: 767px) {
+  h1 {
+    color: yellow;
+    animation: none;
+  }
+}
+
+/*** CONTENT: SIGNUP FORM AND ACCORDION ***/
+
+.content-container {
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  padding: 40px 20px 40px 20px;
+} 
+
+h2 {
+  font-family: 'Permanent Marker', cursive;
+  font-size: 40px;
+  color: rgb(134, 125, 125);
+  text-align: center;
+}
+
+.content-item {
+  width: 40%;
+}
+
+@media (max-width: 1023px) {
+  form {
+    margin-bottom: 40px;
+  }
+  .accordion-container {
+    text-align: center;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .content-item {
+    width: 60%;
+  }
+}
+
+@media (max-width: 767px) {
+  .content-item {
+    width: 80%;
+  }
+}
+
+/*** SIGNUP FORM ***/
+
+form {
+  height: 500px;
+  padding: 20px 40px 20px 40px;
+  border-radius: 20px;
+  box-shadow: 0 4px 8px 4px rgba(0, 0, 0, 0.2);
+  font-family: 'Open Sans', sans-serif;
+}
+
+input[type=text]  {
+  width: 100%;
+  margin: 4px 0 8px 0;
+  padding: 8px;
+  border: none;
+  border-radius: 20px;
+  background-color: rgb(240, 238, 238);
+}
+
+input[type=text]:focus, input[type=e-mail]:focus {
+  outline: none;
```

Comment:
Be aware that the `focus` is there for accessibility reasons. We will talk more about accessibility so no worries :)

Reply:
Is outline-style: none the way to go here to achieve the styling but keep accessibility?

Reply:
Here are some suggestions on how to deal with it: https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/

Reply:
A late thank you, what a great short text about it!